cms_delete_content/README.md

```diff
@@ -0,0 +1,113 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
```

Comment:
Yes, very sure. Odoo admits that file extensions, but actually parses all as RST. Better that they only admit *.rst extension IMO.